deps/build.jl

```diff
@@ -4,7 +4,7 @@ using BinaryProvider # requires BinaryProvider 0.3.0 or later
 const verbose = "--verbose" in ARGS
 const prefix = Prefix(get([a for a in ARGS if a != "--verbose"], 1, joinpath(@__DIR__, "usr")))
 products = [
-    LibraryProduct(prefix, ["libgsl."], :libgsl),
+    LibraryProduct(prefix, ["libgsl"], :libgsl),
```

Comment:
May this be wrong? Was introduced in https://github.com/JuliaMath/GSL.jl/commit/b24af58a4075a9e9c0a14b424d85eb5d808373a8 to fix an issue

Reply:
Before the change [the builds failed completely](https://ci.appveyor.com/project/simonbyrne/gsl-jl/builds/19644659). I'm not sure what the deal with that is.

Reply:
@staticfloat any ideas what is going on here? if we have `"libgsl."` it fails on Windows, but if use `"libgsl"` it fails on 0.6 linux.

Reply:
I added the dot as a hack to stop BinaryProvider from identifying `libgslcblas.so` as the product, rather than `libgsl.so`

https://github.com/JuliaPackaging/BinaryProvider.jl/issues/133